.asf.yaml

```diff
@@ -42,55 +42,27 @@ github:
     # disable rebase button:
     rebase:  false
   protected_branches:
-#    master:
-#      required_status_checks:
+    master:
+      required_status_checks:
         # strict means "Require branches to be up to date before merging".
-#        strict: false
+        strict: false
         # Contexts are the names of checks that must pass.
         # See ./github/workflows/README.md for more documentation on this list.
-#        contexts:
-#          - build
-#          - cpp-tests
-#          - Changed files check
-#          - Build and License check
-#          - CI - Unit - Brokers - Broker Group 1
-#          - CI - Unit - Brokers - Broker Group 2
-#          - CI - Unit - Brokers - Broker Group 3
-#          - CI - Unit - Brokers - Client Api
-#          - CI - Unit - Brokers - Client Impl
-#          - CI - Unit - Other
-#          - CI - Unit - Proxy
-#          - Build Pulsar java-test-image docker image
-#          - CI - Integration - Backwards Compatibility
-#          - CI - Integration - Cli
-#          - CI - Integration - Messaging
-#          - CI - Integration - Shade on Java 8
-#          - CI - Integration - Shade on Java 11
-#          - CI - Integration - Shade on Java 17
-#          - CI - Integration - Standalone
-#          - CI - Integration - Transaction
-#          - Build Pulsar docker image
-#          - CI - System - Function
-#          - CI - System - Pulsar Connectors - Process
-#          - CI - System - Pulsar Connectors - Thread
-#          - CI - System - Pulsar IO
-#          - CI - System - Schema
-#          - CI - System - Tiered FileSystem
-#          - CI - System - Tiered JCloud
-#          - CI - System - Sql
+        contexts:
+           - pulsar-ci-checks-completed
+           - cpp-tests
```

Comment:
I don't find the related step. How can it be the correct name?

Reply:
https://github.com/apache/pulsar/blob/8c7dc4e26f9d590cd8006092c5541c8c655ed33d/.github/workflows/pulsar-ci.yaml#L838

https://github.com/apache/pulsar/blob/8c7dc4e26f9d590cd8006092c5541c8c655ed33d/.github/workflows/ci-cpp.yaml#L38